src/main/java/jenkins/plugin/randomjobbuilder/RandomJobBuilder.java

```diff
@@ -69,7 +69,7 @@ public long getRecurrencePeriod() {
 
         @Override
         protected void doRun() throws Exception {
-            DescriptorImpl d = Jenkins.getInstance().getDescriptorByType(DescriptorImpl.class);
```

Comment:
Findbugs whining, can remove once the current core version is ancient history (and the getInstance that plays like getActiveInstance is now the norm)